src/tree.rs

```diff
@@ -2,15 +2,14 @@ use rand::RngCore;
 
 /// The root of an Orchard commitment tree.
 #[derive(Clone, Debug)]
-pub struct Anchor;
+pub struct Anchor(pub [u8; 32]);
 
 #[derive(Debug)]
 pub struct MerklePath;
 
 impl MerklePath {
     /// Generates a dummy Merkle path for use in dummy spent notes.
-    pub(crate) fn dummy(rng: &mut impl RngCore) -> Self {
-        // TODO
-        MerklePath
+    pub(crate) fn dummy(_rng: &mut impl RngCore) -> Self {
+        todo!()
```

Comment:
This will cause the builder test to fail (and is probably a rebase artifact, as this is what an earlier version of the builder PR contained).